README.md

```diff
@@ -1,2 +1,2 @@
-# RoadFighterCompose
-![](https://github.com/kaiwalyakhasnis/RoadFighterCompose/blob/master/output.gif)
+# RoadFighterCompose
+![](https://github.com/kaiwalyakhasnis/RoadFighterCompose/blob/master/output.gif =250x250)
```

Comment:
Can you change gif size to 480x800 instead?

Reply:
Solution does not work, can you use this format for gif instead

<img src=“https://user-images.githubusercontent.com/16319829/81180309-2b51f000-8fee-11ea-8a78-ddfe8c3412a7.png” width=“150" height=“280”>

Use size as 480x800